NTumbleBit/ClassicTumbler/CycleParameters.cs

```diff
@@ -160,14 +160,14 @@ public CyclePeriods GetPeriods()
 			periods.TumblerChannelEstablishment = new CyclePeriod(tchannelRegistrationStart, tchannelRegistrationEnd);
 			periods.TumblerCashout = new CyclePeriod(tcashoutStart, tcashoutEnd);
 			periods.Payment = new CyclePeriod(paymentStart, paymentEnd);
-			periods.ClientCashout = new CyclePeriod(ccashoutStart, ccashoutEnd);
-			periods.Total = new CyclePeriod(Start, ccashoutEnd + SafetyPeriodDuration);
+			periods.ClientCashout = new CyclePeriod(ccashoutStart, ccashoutEnd + SafetyPeriodDuration);
```

Comment:
Even if this does resolve one particular issue, I am not sure we should be messing with the cycle definitions. The safety period is there for a reason, after all. Since this is at the end of the cycle I suppose the impact is less overall, but I would still advise caution.

Reply:
This change have been reverted